### ISSUE TITLE
Add an option to hide bot names

### ISSUE THREAD

**Issue by GoogleCodeExporter** [janisozaur/androminion#493]
```
After playing a lot of games against AIs, I find that I often tailor my 
strategy according to the opponent I'm playing. This is most notable when 
playing against the Big Money strategy (i.e., Earl). It would be nice to have a 
configuration option to hide the AI names. During the game, the opponents would 
then show as "Player X", and only after the game the identity of the AIs would 
be revealed.
```

Original issue reported on code.google.com by `michal.f...@gmail.com` on 11 Nov 2012 at 10:08
